_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+---
+
+layout: post
+
+title: "A Visual Exploration of EIA Coal Statistics"
+
+author: "Chris Davis"
+
+comments: false
+---
+
+<script type="text/javascript" async
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
+</script>
+
+# Introduction
+
+In the US, coal tends to be quite a [divisive political topic](https://www.economist.com/united-states/2014/05/15/the-politics-of-coal){:target="_blank"}, where discussions can be driven more by ideology than by data.  In this post, we take more of a nuanced approach and dive into detailed public data related to electricity generation from coal, and show how this can be used to better understand the nature of this system and the changes that have been happening.
+
+We investigate aspects related to the life cycle of coal, from its origin in coal mines, to its transport within and between regions, and on to its final use in coal-fired power plants.
+
+Unless otherwise indicated, the analysis below only examines coal use for electricity generation.  While coal can be used for other purposes such as steel production, the US electric power sector is responsible for about [90% of overall consumption of coal](https://www.eia.gov/coal/production/quarterly/pdf/t32p01p1.pdf){:target="_blank"}.
+
+# Data Sources
+
+The data used is published by the [U.S. Energy Information Administration](https://www.eia.gov/) and is publicly available.  In particular, we use several of the [bulk data files](https://www.eia.gov/opendata/bulkfiles.php){:target="_blank"} that have been released as part of their [Open Data API](https://www.eia.gov/opendata/){:target="_blank"}.
+
+The raw data is available in a JSON format, and is very detailed  with information on shipments from specific coal mines to power plants, and also on the operating statistics of power plants.
+
+# Coal Production
+
+Here we use the [EIA bulk data on coal](http://api.eia.gov/bulk/COAL.zip){:target="_blank"} which [can also be browsed online](https://www.eia.gov/opendata/qb.php?category=717234){:target="_blank"}.
+
+The animation below shows the yearly amount of coal produced, both at the level of individual mines, and also aggregated statistics for total mine production per state.  From this one can see that the mines tend to be locally concentrated based on the [various coal basins](https://www.eia.gov/energyexplained/index.php?page=coal_where){:target="_blank"}, and also the amount of production is by no means evenly distributed.
```

Comment:
--> "We can see that the mines tend to be locally concentrated in the [various coal basins](https://www.eia.gov/energyexplained/index.php?page=coal_where){:target="_blank"}, and the amount of production is by no means evenly distributed."